app/views/coronavirus_local_restrictions/_tier_four_restrictions.html.erb

```diff
@@ -0,0 +1,50 @@
+<% tier_heading = capture do %>
+  <%= t("coronavirus_local_restrictions.results.level_four.heading_pretext") %>
+  <span class="coronavirus-local-restriction__heading-tier-label">
+    <%= t("coronavirus_local_restrictions.results.level_four.heading_tier_label") %>
+  </span>
+<% end %>
+
+<%
+  title = if restriction.present? && restriction.future.present?
+    t("coronavirus_local_restrictions.results.changing_levels_title")
+  else
+    tier_heading
+  end
+%>
+
+<% content_for :title, strip_tags(title) %>
+
+<%= tag.div :data => {
+  :module => "coronavirus-track-local-restriction-results",
+  "tracking-label" => "Tier four: #{restriction.area_name}"
+} do %>
+  <%= render "govuk_publishing_components/components/title", {
+    title: sanitize(title)
+  } %>
+
+  <p class="govuk-body">
+    <%= t("coronavirus_local_restrictions.results.match") %> <strong><%= postcode %></strong> to <strong><%= restriction.area_name %></strong>.
+  </p>
+
+  <% if restriction.present? && restriction.future.present? %>
+    <%= render "govuk_publishing_components/components/heading", {
+      text: sanitize(tier_heading),
+      font_size: "m",
+      margin_bottom: 4
+    } %>
+    <p class="govuk-body">
+      <%= t("coronavirus_local_restrictions.results.level_four.changing_alert_level", area: restriction.area_name) %>
+    </p>
+  <% end %>
+
+  <%= render "govuk_publishing_components/components/button", {
+    text: t("coronavirus_local_restrictions.results.guidance_label", level: 4),
+    href: t("coronavirus_local_restrictions.results.level_four.guidance_link"),
+    margin_bottom: 9
+  } %>
+
+  <%= render partial: "coronavirus_local_restrictions/future_restrictions", locals: { restriction: restriction } %>
+  <%= render partial: "coronavirus_local_restrictions/christmas_rules" %>
```

Comment:
I think Christmas rules don't apply to tier 4.

Reply:
So this doesn't matter any more. We'll have some work to do to introduce some logic for the other tiers once that goes live again.